src/manufacturer/manufacturerLayout.component.test.tsx

```diff
@@ -60,3 +65,70 @@ describe('Manufacturer Layout', () => {
     expect(mockedUseNavigate).toHaveBeenCalledWith('/manufacturers');
   });
 });
+
+describe('Manufacturer Layout Error Component', () => {
+  let user: UserEvent;
+
+  beforeEach(() => {
+    user = userEvent.setup();
+  });
+  const createView = () => {
+    return renderComponentWithRouterProvider(
+      <ManufacturerLayoutErrorComponent />
+    );
+  };
+
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('renders manufacturer error page correctly', async () => {
+    const view = createView();
+
+    await waitFor(() => {
+      expect(
+        screen.getByRole('button', {
+          name: 'navigate to manufacturers home',
+        })
+      ).toBeInTheDocument();
+    });
+
+    const homeButton = screen.getByRole('button', {
+      name: 'navigate to manufacturers home',
+    });
+
+    await user.click(homeButton);
+
+    expect(mockedUseNavigate).toHaveBeenCalledTimes(1);
+    expect(mockedUseNavigate).toHaveBeenCalledWith('/manufacturers');
+
+    expect(view.asFragment()).toMatchSnapshot();
+  });
+});
+
+describe('manufacturerLayoutLoader', () => {
```

Comment:
This name is inconsistent with the describe above.

Reply:
This is on purpose as this is a function and there other ones are components